doc/api/prev_api_changes/api_changes_3.1.0.rst

```diff
@@ -800,7 +800,7 @@ Changes in parameter names
   `.bezier.split_bezier_intersecting_with_closedpath`,
   ``bezier.find_r_to_boundary_of_closedpath``,
   `.bezier.split_path_inout` and `.bezier.check_if_parallel` has been renamed to
-  *tolerance*.
```

Comment:
`tolerance` is the spelling in the source, please drop this change, e.g. https://github.com/matplotlib/matplotlib/blob/b01462c9ac7b0d3875892d6f40c9213c56f0864f/lib/matplotlib/bezier.py#L113-L114

so this change needs to be reverted.

Reply:
Resolved